.github/workflows/run-tests.yml

```
@@ -0,0 +1,26 @@
+# Run tests when merged to main branch
+
+name: Run all the tests
+on:
+  push:
+    branches:
+      - main
```

```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```
I think we should also run tests in PRs